protocol/messenger_communities.go

```diff
@@ -4067,28 +4067,51 @@ func (m *Messenger) CheckPermissionsToJoinCommunity(request *requests.CheckPermi
 	return m.communitiesManager.CheckPermissionToJoin(request.CommunityID, addresses)
 }
 
-func (m *Messenger) CheckCommunityChannelPermissions(request *requests.CheckCommunityChannelPermissions) (*communities.CheckChannelPermissionsResponse, error) {
-	if err := request.Validate(); err != nil {
-		return nil, err
-	}
+func (m *Messenger) checkCommunityPermissions(communityID string, sharedAddresses []*protobuf.RevealedAccount, requestAddresses []string) ([]gethcommon.Address, error) {
```

Comment:
I think this function name doesn't really make sense. We're not really checking anything here, just getting the right addresses right?